helm/botkube/templates/deployment.yaml

```diff
@@ -31,7 +31,7 @@ spec:
           volumeMounts:
             - name: config-volume
               mountPath: "/config"
-          {{- if ne .Values.config.communications.mattermost.cert "None" }}
+          {{- if ne .Values.config.ssl.cert "None" }}
```

Comment:
1. With the suggested change in the `values.yaml`, we can replace this with
```suggestion
          {{- if .Values.config.ssl.cert }}
```

Reply:
Due to my comment above about the templates/secret.yaml file, used this instead
```suggestion
          {{- if .Values.config.ssl.enabled }}
```